javascript-constructors/example-constructor.js

```diff
@@ -0,0 +1,10 @@
+function ExampleConstructor() {
+  console.log('value of prototype:', ExampleConstructor);
+  console.log('typeof prototype:', typeof ExampleConstructor);
+}
```

Comment:
Actually we want to console log value and type of prototype property of `ExampleConstructor`.

Also, we want to leave `ExampleConstructor` as an empty `{}` and do above outside of the function